_lib/core/common.php

```diff
@@ -548,7 +551,7 @@ function send_mail($opts = []): bool
     $is_html = ($opts['content'] !== strip_tags($opts['content']));
     
     if (getenv('SENDGRID_API_KEY')) {
-        $email = new SendGrid\Mail\Mail();
+        $email = new Mail();
```

Comment:
this will break things as I removed the namespace

Reply:
The import statement is back at the top of the file, is that what you mean?

Reply:
I think I've restored this?